examples/differentially_private_sgd.py

```diff
@@ -138,7 +138,7 @@ def clipped_grad(params, l2_norm_clip, single_example_batch):
   nonempty_grads, tree_def = tree_flatten(grads)
   total_grad_norm = jnp.linalg.norm(
       [jnp.linalg.norm(neg.ravel()) for neg in nonempty_grads])
-  divisor = jnp.max((total_grad_norm / l2_norm_clip, 1.))
+  divisor = jnp.max(jnp.array((total_grad_norm / l2_norm_clip, 1.)))
```

Comment:
How about this instead?

```suggestion
  divisor = jnp.maximum(total_grad_norm / l2_norm_clip, 1.)
```